### DISCUSSION TITLE
What is the varaible "trame__busy"?

### DISCUSSION THREAD

**Discussion by Jimmy-KL** [Kitware/trame#163] [Q&A] [answered]
I didn't see the definition of the variable "trame__busy" in the following code. Is it a global variable defined at some where else? What is the function of the variable?
` vuetify.VProgressLinear(
            indeterminate=True,
            absolute=True,
            bottom=True,
            active=("trame__busy",),
        )`

**Comment by jourdain** [comment, marked as answer]
You can look at the [reserved-state-entries](https://github.com/Kitware/trame#reserved-state-entries) of the readme.
